javascript/javascript1/week4/voice_assistant.js

```diff
@@ -0,0 +1,86 @@
+let userName;
+const todo = [];
+
+function getReply(command) {
+  if (typeof command === 'string') {
+    command = command.toLowerCase();
+  }
+  if (command.includes('hello my name is')) {
+    const words = command.split(' ');
+    userName = words[words.length - 1];
+    return `Nice to meet you ${userName}`;
+  } else if (command === 'what is my name') {
+    if (!userName) {
+      return 'please introduce yourself';
+    } else {
+      return userName;
+    }
+  } else if (command.includes('to my todo') && command.includes('add')) {
+    const taskToAdd = command.substring(
+      command.indexOf('add') + 4,
+      command.lastIndexOf(' to my todo')
+    );
+    todo.push(taskToAdd);
+    return `${taskToAdd} added to your todo`;
+  } else if (command.includes('from my todo') && command.includes('remove')) {
+    const taskToRemove = command.substring(
+      command.indexOf('remove') + 'remove '.length,
+      command.lastIndexOf(' from my todo')
+    );
+    for (const item of todo) {
+      if (item === taskToRemove) {
+        todo.splice(todo.indexOf(item), 1);
+        return `Removed ${taskToRemove} from your todo`;
+      }
+    }
+  } else if (command === 'what is on my todo?') {
+    if (todo.length === 1) {
+      return `You have 1 todo - ${todo[0]}`;
+    } else if (todo.length === 1) {
```

Comment:
This is the same condition as the if statement

Reply:
I've fixed it. Thank you for reviewing! 🙌